help/C/index.docbook

```diff
@@ -1177,20 +1133,20 @@
         </orderedlist>
       </sect3>
 
-      <sect3 id="mate-system-monitor-customize-mem">
-        <title>To Change the Line Colors of the Memory and Swap Graph</title>
+      <sect3 id="mate-system-monitor-customize-network">
+        <title>To Change the Line Colors of the Network Graph</title>
         <para>
-          To change the color of the lines that represent memory and swap usage in the graph, perform the following steps:
+          To change the background color of the <application>&app;</application> graphs, perform the following steps:
```

Comment:
Tip: Try to use the same text displayed in the window. It increases the user comprehension.
```
<title>To Change the Line Colors of the Network History Graph</title>
...
To change the line colors of the <guilabel>Network History</guilabel> graph, ...
```